core/src/test/resources/org/apache/calcite/test/SqlToRelConverterTest.xml

```diff
@@ -4404,6 +4404,29 @@ LogicalProject(D2=[$0], D3=[$1])
                     LogicalFilter(condition=[=($1, $0)])
                       LogicalProject(D4=[+($0, 4)], D5=[+($0, 5)], D6=[+($0, 6)])
                         LogicalTableScan(table=[[CATALOG, SALES, DEPT]])
+]]>
```

Comment:
I checked a good number of the more complex nested queries by eyeball check, as the plan generated when handling nested sub queries gets very complicated very quickly.

Reply:
I'm going to trust these are correct. Please let me know if you need me to review any by hand.